crates/node-bindings/src/anvil.rs

```diff
@@ -375,6 +384,19 @@ mod tests {
         let _ = Anvil::new().arg("--accounts").arg("20").spawn();
     }
 
+    #[test]
+    fn assert_block_time_is_natural_number() {
+        //This test is to ensure that older versions of anvil are supported
+        //even though the block time is a f64, it should be passed as a whole number
+        let anvil = Anvil::new().block_time(12);
+        assert_eq!(anvil.block_time.unwrap().to_string(), "12");
```

Comment:
Should also test that it spawns correctly
```suggestion
        assert_eq!(anvil.block_time.unwrap().to_string(), "12");
        let _ = anvil.spawn();
```